core/notifications/proto/notifications.proto

```diff
@@ -237,19 +237,49 @@ message PriceChanged {
 
 message MarketingNotificationTriggered {
   repeated string user_ids = 1;
-  map<string, LocalizedPushContent> localized_push_content = 2;
+  map<string, LocalizedContent> localized_content = 2;
   optional DeepLink deep_link = 3;
+  bool should_send_push = 4;
+  bool should_add_to_history = 5;
+  bool should_add_to_bulletin = 6;
 }
 
-message LocalizedPushContent {
+message LocalizedContent {
   string title = 2;
   string body = 3;
```

Comment:
why 2, 3 instead of 1,2 .. . This is a breaking change so might as well fix this also

Reply:
Yeah thats a good point, I will fix it